### PR TITLE
Adding coverage for make_user method

### LINKED ISSUE #1714: Add CLI tests for two new parameters in factory.make_user
The two new parameters are:

```
--default-location-id        DEFAULT_LOCATION_ID
--default-organization-id    DEFAULT_ORGANIZATION_ID
```

### PR DESCRIPTION
**Pull request by oshtaier**
Cover two more parameters in order to
Close #1714 issue

### CODE REVIEW

**Comment by tkolhar** [comment]
ACK

File: tests/foreman/cli/test_user.py [hunk 1]
```diff
@@ -247,6 +253,36 @@ def test_positive_create_user_6(self):
...
+        """@Test: Check if user with location can be created
+
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new location assigned
+
+        """
+        try:
+            location = make_location()
+            user = make_user({'location-ids': location['id']})
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: According to #1714 the option that should be used here is `--default-location-id`

File: tests/foreman/cli/test_user.py [hunk 2]
```diff
@@ -247,6 +253,36 @@ def test_positive_create_user_6(self):
...
+        """@Test: Check if user with organization can be created
+
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new organization assigned
+
+        """
+        try:
+            org = make_org()
+            user = make_user({'organization-ids': org['id']})
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: According to #1714 the option that should be used here is `--default-organization-id`.

File: tests/foreman/cli/test_user.py [hunk 3]
```diff
@@ -247,6 +253,36 @@ def test_positive_create_user_6(self):
...
+
+        @Assert: User is created and has new location assigned
+
+        """
+        try:
+            location = make_location()
+            user = make_user({'location-ids': location['id']})
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.assertIn(location['name'], user['locations'])
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: Maybe the assertion should be updated to check for the default location information

File: tests/foreman/cli/test_user.py [hunk 4]
```diff
@@ -247,6 +253,36 @@ def test_positive_create_user_6(self):
...
+
+        @Assert: User is created and has new organization assigned
+
+        """
+        try:
+            org = make_org()
+            user = make_user({'organization-ids': org['id']})
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.assertIn(org['name'], user['organizations'])
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: Maybe the assertion should be updated to check for the default organization information

File: tests/foreman/cli/test_user.py [hunk 5]
```diff
@@ -247,6 +253,50 @@ def test_positive_create_user_6(self):
...
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new default organization assigned
+
+        @BZ: 1213426
+
+        """
+        try:
+            org = make_org()
+            user = make_user(
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: please format dicts with more than one element like the following:

```python
user = make_user({
    'a': 'b',
    'c': 'd',
})
```

Be aware of the trailing `,` on the last line, it is required in our style convention.

File: tests/foreman/cli/test_user.py [hunk 6]
```diff
@@ -247,6 +253,50 @@ def test_positive_create_user_6(self):
...
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new default location assigned
+
+        @BZ: 1213426
+
+        """
+        try:
+            location = make_location()
+            user = make_user(
```
Comment (elyezer) [inline tests/foreman/cli/test_user.py]: same as https://github.com/SatelliteQE/robottelo/pull/2084/files#r28705154

**Comment by elyezer** [comment]
APT

**Pull request merged**